agenta-backend/agenta_backend/services/organization_service.py

```diff
@@ -33,3 +55,67 @@ class NotFound(Exception):
     """Custom exception for credentials not found"""
 
     pass
+
+
+def send_invitation_email(
+    email: str, token: str, organization: Organization, user: User
+):
+    message = Mail(
+        from_email="account@hello.agenta.ai",
+        to_emails=email,
+        subject=f"{user.username} invited you to join {organization.name}",
+        html_content=f"<p>Hello,</p>"
+        f"<p>{user.username} has invited you to join {organization.name} on Agenta. "
+        f"Click the link below to accept the invitation:</p>"
+        f'<a href="https://demo.agenta.ai/organisations/accept?token={token}&org_id={organization.id}">Accept Invitation</a>',
+    )
+    try:
+        sg.send(message)
+        return True
+    except Exception as e:
+        print(e)
```

Comment:
Printing the exception is good for debugging, sure. What about the user? Shouldn't they know that the invitation did not go through and needs to try again?

Reply:
Oh yes. Thank you, I missed that.